rest-server/src/models/job.js

```diff
@@ -276,6 +276,25 @@ class Job {
       });
   }
 
+  executeJob(name, data, next) {
```

Comment:
>executeJob [](start = 2, length = 10)

putJobExecutionType would be a better name.

Reply:
updated